### DISCUSSION TITLE
Auto Zap

### DISCUSSION THREAD

**Discussion by ghost** [stackernews/stacker.news#1877] [Ideas]
**Is your feature request related to a problem? Please describe.**
Stackers have a number of methods available for zapping content on Stacker News. They can click the Lightning bolt, long-press for a custom amount, or enable turbo zapping in their settings.

However, all of these methods require at least one click for a zap to occur. What if stackers could zap great posts and comments without any extra clicks at all?

**Describe the solution you'd like**
To lower the barrier for stackers to zap more, Stacker News should offer an opt-in zap setting called "AutoZap" or something similar. Instead of stackers manually zapping content they like, this setting would automatically zap any posts a stacker reads (by either visiting the post page or clicking the post link) as well as zapping any posts or comments they reply to.

This setting should inherit the default zap amount set by each stacker.

This feature request is inspired by this comment: https://stacker.news/items/391517

**Comment by benthecarman** [comment]
Something that could be interesting would be for the amount of time you spend on the post.
